src/main/java/com/minecolonies/coremod/colony/managers/ColonyPackageManager.java

```diff
@@ -129,6 +129,11 @@ else if (lastContactInHours != 0)
             {
                 @NotNull final EntityPlayerMP player = (EntityPlayerMP) o;
 
+                if (player.connection.networkTickCount < 5)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 5 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)